### PR TITLE
I made your chat.md stub into a thicker stub

### PR DESCRIPTION
**Pull request by equanimityspace**
Just made it readable, added link to co lookup. Not exactly sure what else you're looking for here, but I can add it later if you want.

Also, the link may be scuffed, I definitely winged it.

### CODE REVIEW

File: docs/staff-guide/chat.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Chat
 
-/chatlock
\ No newline at end of file
+Chat can be locked and unlocked with `/chatlock`.
+
+[//]: # (I'm just assuming it can be unlocked with the same command here)
+
+To check past chat messages, use [core protect's logs.](/docs/staff-guide/core.md/#lookup)
```
Comment (eartharoid) [inline docs/staff-guide/chat.md]:
```suggestion
To check past chat messages, use [core protect's logs.](/staff-guide/core#lookup)
```

**Review by eartharoid — CHANGES_REQUESTED** [review]

File: docs/staff-guide/chat.md (hunk 1 above)
Comment (equanimityspace) [reply docs/staff-guide/chat.md]: `/staff-guide/core/#lookup` actually works, but I'll change it for the sake of consistency

Comment (eartharoid) [reply docs/staff-guide/chat.md]: CoreProtect chat lookup does work (which surprised me a little), but it only shows messages sent on the survival server and it's much easier to use Discord for this (easy to scroll, has search etc).

Comment (equanimityspace) [reply docs/staff-guide/chat.md]: Oh shoot, true. I'm a boomer so before discord that's what I used for chat logs. And it's interesting that it doesn't log messages from the creative server, though its not surprising.

**Comment by equanimityspace** [comment]
looks & works as intended finally

**Pull request merged**